### ISSUE TITLE
vendored_libraries doesn't copy .dylib files in Frameworks

### ISSUE THREAD

**Issue by orestis** [CocoaPods/CocoaPods#3577]
I have a private pod that needs to vendor (and bundle) both a .framework file and a .dylib library. This is OSX. Let's assume the framework filename is 'MyFramework.framework' and the dylib is 'libMyDylib.dylib'. Both reside and a 3rdParty directory.

I would expect any of this to work:
```
  s.vendored_frameworks = '3rdParty/MyFramework.framework' 
  s.vendored_libraries = '3rdParty/libMyDyLib.dylib'
```
```
  s.vendored_frameworks = '3rdParty/MyFramework.framework' 
  s.vendored_libraries = '3rdParty/libMyDyLib'
```
```
  s.vendored_frameworks = '3rdParty/MyFramework.framework' 
  s.vendored_libraries = '3rdParty/MyDyLib'
```

None of those work. The framework does get placed under Frameworks, the dylib is not. My workaround is this:

```
  s.resources = ['3rdParty/libMyDylib.dylib', '3rdParty/MyFramework.framework']
```

And also I have to tweak LD_RUNPATH_SEARCH_PATHS and LD_RUNPATH_SEARCH_PATHS to point to get everything working.

**Comment by neonichu** [comment]
The requirement to tweak linker settings is a known issue, see #1993

**Comment by segiddins** [comment]
This should be fixed in the 0.39 beta.
